qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -472,6 +472,76 @@ def parse_offset_get(s):
         self._buffer2_ready = False
 
         self.connect_message()
+    
+    
+    SNAP_PARAMETERS = {
+            'x': '1',   
+            'y': '2',  
+            'r': '3', 
+            'p': '4',   
+        'phase': '4',  
+           'θ' : '4',
+         'aux1': '5',  
+         'aux2': '6',   
+         'aux3': '7',  
+         'aux4': '8',  
+         'freq': '9',   
+          'ch1': '10',
+          'ch2': '11'  
+    }
+    
+    def snap(self, *parameters: str) -> Tuple[float, ...]:
```

Comment:
I think `Tuple` was never imported, which, as far as I can tell, is the only reason why the CI is currently failing. Just adding `from typing import Tuple` to the imports should fix that.